src/nav/components.yml

```diff
@@ -89,7 +89,7 @@
     - title: Tabs
       id: /components/tabs
     - title: Tags
-      id: https://zendeskgarden.github.io/react-components/tags
+      id: /components/tags
```

Comment:
Curious if you searched the website for opportunities to connect `/components/tags` reference links? On the [Multiselect](https://garden.zendesk.com/components/multiselect) page, for example.

Reply:
Wow, great call. This should be procedure for every PR merge.

Reply:
I've been thinking of adding it as an item to the [PR Checklist](https://github.com/zendeskgarden/website/blob/main/.github/PULL_REQUEST_TEMPLATE.md) :wink: